4-projects/modules/infra_pipelines/main.tf

```diff
@@ -66,13 +68,30 @@ resource "google_storage_bucket" "cloudbuild_artifacts" {
   }
 }
 
+resource "google_storage_bucket" "cloudbuild_bucket" {
```

Comment:
This seems like we could refactor to use for_each over `merge(local.artifact_buckets, local.state_buckets, local.cloudbuild_bucket)`?